### PR TITLE
0 0 17 release

### PR DESCRIPTION
**Pull request by christiam**
Notable changes:
* The quickstart became overview and links to cloud vendor specific quickstarts
* The minimal configuration parameters for AWS were updated
* Several AWS configuration parameters were marked as optional

### CODE REVIEW

File: docs/source/quickstart-aws.rst [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+**Note**: this command requires the `AWS CLI SDK <https://aws.amazon.com/cli/>`_.
+
+.. code-block:: bash
+
+    aws s3 cp ${YOUR_RESULTS_BUCKET}/*.out.gz . 
+
+Clean up
+--------
+This step is **critical**, please do not omit it, even if you ran Ctrl-C when
+starting ElasticBLAST. If you do not clean up your cloud resources, you will accrue charges from
```
Comment (boratyng) [inline docs/source/quickstart-aws.rst:147]: The bit about charges is not true for AWS.

**Review by boratyng — DISMISSED** [review]

File: docs/source/quickstart-aws.rst (hunk 1 above)
Comment (christiam) [reply docs/source/quickstart-aws.rst:147]: Thanks for the feedback. Please see 2.a. here: https://confluence.ncbi.nlm.nih.gov/display/EB/2020-10-13+Backlog+grooming

**Review by tom6931 — DISMISSED** [review]
Few comments:

Why is the quickstart example for AWS different than the one for GCP?  My bias is that pdbnt is pretty much useless as an example, but it is also missing the BLAST options. The options are useful as an example but also since the user will, probably to their surprise, end up with BLAST archive format that they then need to deal with.

It would be good to mention the Requirements page on the quickstart page for each provider.  It would be a step to review the requirements for the platform.  This came up today when I was working with Rich, and I didn't see kubectl.  Of course, I just had to look off a bit.

In general, it seems we should strongly encourage users to run their cluster where we host databases.  Is there any downside to doing that?  E.g., I'm doing everything in Iowa, but run my cluster in N. Va.  Better to drag the queries across the country than the databases.  This may be a topic for a longer discussion.

Good idea to break up the quickstarts though.

**Comment by christiam** [comment]
> Few comments:
> 
> Why is the quickstart example for AWS different than the one for GCP? 

Primarily because `blastp` hasn't been tested well enough, but I'll change the example nonetheless.

> It would be good to mention the Requirements page on the quickstart page for each provider.

This is done.

> In general, it seems we should strongly encourage users to run their cluster where we host databases.

Agreed, and this is already done: 
https://elbdoc--29.org.readthedocs.build/en/29/configuration.html#elb-aws-region
https://elbdoc--29.org.readthedocs.build/en/29/configuration.html#gcp-region

**Review by boratyng — APPROVED** [review]

**Pull request merged**